src/plugins/data/public/index_patterns/fields/index.ts

```diff
@@ -17,5 +17,4 @@
  * under the License.
  */
 
-export * from './field_list';
-export * from './field';
+export * from '../../../common/index_patterns/fields';
```

Comment:
Rather than re-exporting here, should we just remove the `public/index_patterns/fields` directory entirely and import items from `common/index_patterns/fields` wherever they are needed?

Reply:
These haven't been removed yet because I ran into errors when I did them all at once. There's an undesirable import somewhere. Just a matter of chasing it down now or later.